docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
Can we include all the docker-specific build stuff here so that we don't break the Makefile for non-linux platforms?